packages/esm-patient-chart-app/src/visit/visit-dialog.component.tsx

```diff
@@ -1,24 +1,18 @@
 import React from 'react';
-import { useVisitDialog } from './useVisitDialog';
-import StartVisitPrompt from './visit-prompt/start-visit.component';
-import EndVisitPrompt from './visit-prompt/end-visit.component';
+import StartVisit from './visit-prompt/start-visit.component';
+import EndVisit from './visit-prompt/end-visit.component';
+import CancelVisit from './visit-prompt/cancel-visit.component';
 
 interface VisitDialogProps {
   patientUuid: string;
 }
 
-const VisitDialog: React.FC<VisitDialogProps> = ({ patientUuid }) => {
-  const { type, state } = useVisitDialog(patientUuid);
-  const closeModal = () => {
-    window.dispatchEvent(new CustomEvent('visit-dialog', { detail: { type: 'close' } }));
-  };
-
-  return (
-    <>
-      <StartVisitPrompt isModalOpen={type == 'prompt'} closeModal={closeModal} state={state} />
-      <EndVisitPrompt isModalOpen={type === 'end'} patientUuid={patientUuid} closeModal={closeModal} />
-    </>
-  );
-};
+const VisitDialog: React.FC<VisitDialogProps> = ({ patientUuid }) => (
+  <>
+    <StartVisit patientUuid={patientUuid} />
+    <EndVisit patientUuid={patientUuid} />
+    <CancelVisit patientUuid={patientUuid} />
```

Comment:
Two nits, totally optional:
1. This looks like 3 components stacked on top of one another. I think checking the `type` and only mounting the relevant one would be preferable in terms of code clarity.
2. I'd call each of those `StartVisitDialog`, `EndVisitDialog`, etc.